### ISSUE TITLE
Suggestion: log precision

### ISSUE THREAD

**Issue by Songbird0** [TheHolyWaffle/TeamSpeak-3-Java-API#219]
Hi!

When I get an error sent by TeamSpeak server, its logs aren't really accurate.

For instance:
```
[2017-07-05 18:59:11.769] [SEVERE] Command channellist was not answered in time.
```
There's the date and hour, but the class and subprocess aren't specified.

I suggest to use the standard [`Logger`](https://docs.oracle.com/javase/8/docs/api/java/util/logging/Logger.html) class. `Logger` allows to print all needed details to debug a program efficiently.

Example:

```java
public class HelloWorld
{
  // arguments are passed using the text field below this editor
  public static void main(String[] args)
  {
    final Logger log = Logger.getLogger(HelloWorld.class.getName());
    final String foo = "foo", bar = "bar", baz = "baz";
    log.severe(String.format("some informations:\n%1$s\n%2$s\n%3$s", foo, bar, baz));
  }
}
```
Output:
```text
Jul 05, 2017 5:28:58 PM HelloWorld main
                                   ^ subprocess name
SEVERE: some informations:
foo
bar
baz
```

**Note**: Package, if any, will be printed thanks to `getName()` method.

How about that?

Regards.

**Comment by rogermb** [comment]
Hi again!

We already switched to the SLF4J logging facade, which (among other things) also takes care of your feature request. This will be included in the upcoming release of version 1.1.

If you use the `-with-dependencies` artifact, we'll ship the API with the simple SLF4J logging implementation and a default configuration. You will be able to change or override that configuration to include the class name, thread name, etc.

I chose not to include these values by default, though, to mirror the logging format we used in the current release and because I don't really see the point in including them in this rather small project.

If you use the default maven artifact, there won't be a logging provider included, so you'll be free to add a logging implementation and to configure everything by yourself.

**Comment by Songbird0** [comment]
Ok, I'll wait the new version.

> If you use the `-with-dependencies` artifact, we'll ship the API with the simple SLF4J logging implementation and a default configuration.

Have you an example, please?

**Comment by Kakifrucht** [comment]
That would only apply if you download the API and include it via your IDE, instead of using Maven.

**Comment by Songbird0** [comment]
> That would only apply if you download the API and include it via your IDE, instead of using Maven.

Ok, I don't want that.

**Comment by rogermb** [comment]
That's the JAR file you can download in the [GitHub releases page](https://github.com/TheHolyWaffle/TeamSpeak-3-Java-API/releases). You can check our default configuration file [here](https://github.com/TheHolyWaffle/TeamSpeak-3-Java-API/blob/master/src/main/resources/simplelogger.properties).

If you [look up the project on Maven central](https://search.maven.org/#search%7Cga%7C1%7Ca%3A%22teamspeak3-api%22), there will be different artifacts you can download or include in your project by editing your `pom.xml` file: jar, javadoc, sources and with-dependencies.

**Comment by Songbird0** [comment]
Understood, thanks.

**Comment by rogermb** [comment]
Also released in v1.1.0. Sorry about the notification emails 😅
